src/OpenTelemetry.Exporter.Zipkin/Implementation/ZipkinSpan.cs

```diff
@@ -284,15 +285,11 @@ public void Write(Utf8JsonWriter writer)
                 {
                     if (tag.Value is int intValue)
                     {
-                        writer.WriteNumber(tag.Key, intValue);
-                    }
-                    else if (tag.Value is bool boolVal)
-                    {
-                        writer.WriteBoolean(tag.Key, boolVal);
+                        writer.WriteString(tag.Key, intValue.ToString(CultureInfo.InvariantCulture));
                     }
                     else if (tag.Value is double doubleVal)
                     {
-                        writer.WriteNumber(tag.Key, doubleVal);
+                        writer.WriteString(tag.Key, doubleVal.ToString(CultureInfo.InvariantCulture));
                     }
                     else if (tag.Value is string stringVal)
                     {
```

Comment:
Do already have an issue to add support for arrays of the basic types supported by OTel?

Reply:
not that i'm aware. But, let's discuss!
If we receive an array of ints, how should we send to zipkin?

Reply:
The other languages you linked above seem to create. a comma delimited string of values which I think would be the best option

Reply:
yeah, i'm not that familiar with py, but i saw java implementation. @pjanotti , i will add this as well in this pr.

Reply:
@pjanotti , can you check again?

Reply:
This looks good to me.